<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#bgWidget, #centralwidget{
	background-color: rgb(36, 62, 51);
	border: none;
}

#mainContentWidget{
	border-top-right-radius:14px;
	border-bottom-right-radius:14px;
	background-color: rgb(251, 244, 238);
}

#dailyPushButton{
	background-color: rgb(131, 174, 157);
	color: white;
	font: 63 11pt &quot;Yu Gothic UI Semibold&quot;;
}
QPushButton{
	text-align:left;
	padding:7;
	border-top-left-radius:7px;
	border-bottom-left-radius:7px;
}
QListWidget{
	border:None;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="bgWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="sideMenuWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>17</number>
           </property>
           <property name="topMargin">
            <number>65</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>25</number>
           </property>
           <item>
            <widget class="QWidget" name="topMenuWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <property name="spacing">
               <number>14</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="dailyPushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="weeklyPushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="monthlyPushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QWidget" name="bottomMenuWidget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="settingsPushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mainContentWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>14</number>
           </property>
           <property name="topMargin">
            <number>18</number>
           </property>
           <property name="rightMargin">
            <number>14</number>
           </property>
           <property name="bottomMargin">
            <number>18</number>
           </property>
           <item>
            <widget class="QListWidget" name="todoListWidget">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="showDropIndicator" stdset="0">
              <bool>true</bool>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
             <property name="movement">
              <enum>QListView::Static</enum>
             </property>
             <property name="resizeMode">
              <enum>QListView::Fixed</enum>
             </property>
             <property name="sortingEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="inProgressListWidget">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="doneListWidget">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="dragEnabled">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
